Project/Core/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +33,7 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
+[assembly: AssemblyVersion("2.0.0.0")]

```

Comment:
2?

Reply:
yep, it has been changed since last release